scope-key-data.js

```diff
@@ -258,13 +266,17 @@ canReflect.assignSymbols(ScopeKeyData.prototype, {
 	},
 	"can.setPriority": function(newPriority){
 		canReflect.setPriority( this.observation, newPriority );
-	},
-	//!steal-remove-start
-	"can.getName": function() {
+	}
+};
+
+//!steal-remove-start
+if (process.env.NODE_ENV !== 'production') {
+	scopeKeyDataPrototype[canSymbol.for("can.getName")] = function() {
```

Comment:
the canSymbol.for() here is not needed (and might cause it not to work, I'm not sure), because this is already being passed into canReflect.assignSymbols()